docs/using/staking.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Staking
+authors: "Tim McMackin"
+last_update:
+  date: 17 December 2024
+---
+
+Staking is the process of temporarily locking tez on the Tezos platform.
+Staked tez stays in the staker's account, but they cannot transfer that tez while it is staked.
```

Comment:
shouldn't it be plural? "the tez staked" suggests 1 tez, but maybe I'm wrong:
```suggestion
Staked tez stays in the staker's account, but they cannot be spent (e.g., transferred to another account).
```

Reply:
It's tricky, "tez" can be singular or plural. I made this "but the staker cannot transfer (or spend) that tez."